README

```diff
@@ -1 +1,62 @@
-Presto Manager: A tool for managing Presto clusters.
+# Presto Manager
+
+A tool for managing Presto clusters. Targeted for Linux OS, not guarentee to work on Windows/MacOS.
+
+## Getting Started
+
+These instructions will get you a instance of the project up and running on your local machine for development and testing purposes.
+
+### Prerequisites
+
+- JDK 8, available at (http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html)
+         Alternative  (http://openjdk.java.net/install/)
```

Comment:
To improve readability in the plaintext, the links can be placed in link references.

```markdown
- Java Development Kit 8, available at [oracle.com][Oracle Java] or [openjdk.java.net][OpenJDK]

[OpenJDK]: http://openjdk.java.net/install
[Oracle Java]: http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html
```
(The last two lines shoud be near the bottom of the document.)

Though maybe [this page](https://docs.oracle.com/javase/8/docs/technotes/guides/install/install_overview.html) would be better as the Oracle link.